app/controllers/support/HttpPartConfigChecker.scala

```diff
@@ -13,7 +13,11 @@ trait HttpPartConfigChecker {
 }
 
 object HttpPartConfigChecker {
-  def apply(httpPartConfig: HttpPartConfig)(implicit lang: Lang, app: Application): Seq[String] = Seq(QueryParamInterpolation, MissingPathParam, PathParamNoInterp, PathParamOption, WhitespaceInUri, AlertEmailOff(app.mode)).flatMap(_.warnings(httpPartConfig))
+  def apply(httpPartConfig: HttpPartConfig)(implicit lang: Lang, app: Application): Seq[String] = {
+    val checks = Seq(QueryParamInterpolation, MissingPathParam, PathParamNoInterp, PathParamOption, WhitespaceInUri) ++
+      (if (app.configuration.getString("application.env").contains("production")) Seq(AlertEmailOff) else Nil)
```

Comment:
`app.configuration.getString("application.env")` could be cached in a val somewhere, like on `HttpPartConfigChecker`, since Typesafe config is not exactly performant.